common/assets.go

```diff
@@ -110,7 +110,19 @@ var (
 		"XTZ",
 	}
 
-	AllAssets = MergeLists(PEGAsset, CurrencyAssets, CommodityAssets, CryptoAssets, V4CurrencyAdditions, V4CryptoAdditions)
+	V5CryptoAdditions = []string{
+		"HBAR",
+		"NEO",
+		"AED",
```

Comment:
I can't seem to find a reference to this crypto token anywhere. If this is supposed to be the UAE dirham, this should not be added to "CryptoAdditions"

Reply:
Ok, added V5CurrencyAdditions with AED @WhoSoup